src/main/java/am/ik/yavi/constraint/time/ChronoLocalDateConstraintBase.java

```diff
@@ -0,0 +1,37 @@
+package am.ik.yavi.constraint.time;
+
+import am.ik.yavi.core.Constraint;
+import am.ik.yavi.core.ConstraintPredicate;
+
+import java.time.chrono.ChronoLocalDate;
+import java.util.function.Predicate;
+
+import static am.ik.yavi.core.NullAs.VALID;
+import static am.ik.yavi.core.ViolationMessage.Default.*;
+
+/**
+ * This is the base class for constraints on ChronoLocalDate.
+ *
+ * @author Diego Krupitza
+ */
+public abstract class ChronoLocalDateConstraintBase<T, V extends ChronoLocalDate, C extends Constraint<T, V, C>>
+		extends ComparableTemporalConstraintBase<T, V, C> {
+
+	/** Is the given ChronoLocalDate a leap year */
+	public C leapYear() {
```

Comment:
What are the use cases that require this constraint?
I can't imagine a case where I want to check that the input value must be a leap year.

Reply:
TBH I can not really find a use case for this too. I can not remember why I added this constraint. 
I think I added this because `ChronoLocalDate` is the only class that supports "leap year" checks.

Reply:
Let's remove this method. I don't want to add unnecessary methods.

Reply:
Done